docs/installation/auth/nginx.md

```diff
@@ -47,18 +47,37 @@ server {
     location = /favicon.ico { access_log off; log_not_found off; }
 
     location /static/ {
-        alias /var/www/myauth/static;
+        alias /var/www/myauth/static/;
         autoindex off;
     }
 
     # Gunicorn config goes below
     location / {
-        include proxy_params;
```

Comment:
Why? This contains everything defined below.

Reply:
i duped across the ngxinx config for gunicorn from docs/installation/auth/gunicorn.md to the gunicorn related section in docs/installation/auth/nginx.md

it listed the proxy params manually and they conflict with include proxy_params;